src/constants/hwConstants.js

```diff
@@ -0,0 +1,8 @@
+export const hwConstants = {
+  secondPassphraseOffset: 1e5,
```

Comment:
Based on what we have decided here: https://github.com/LiskHQ/lisk-hub/issues/1101 the second passphrase will be disabled for ledger. It means you can remove all the code related to this functionality (also `setSecondPassphraseWithLedger ` )

Reply:
@hirishh Hey, I will contact you when I will have more presentable code via liskchat because so far I'm just pushing whatever works for me.
Thanks for feed back :)

Reply:
Sure! I'm here to help ;)